flight/mcu/lib/teensyShared/rocketFSM.h

```diff
@@ -12,6 +12,14 @@ class rocketFSM {
 
    private:
     pointers *pointer_struct;
+    float linear_acceleration;
+    FSM_State rocket_state;
+    uint32_t launch_time;
+    uint32_t burn_timer;
+    uint32_t burnout_time;
+    uint32_t coast_timer;
```

Comment:
chrt has types systime_t and sysinterval_t for time values. Both are just typedefs for uint32_t though.

Reply:
Is there a different between systime_t and sysinterval_t?

Reply:
they are the same underlying type, the only difference is clarity

Reply:
So sysinterval_t could be used for timers and systime_t could be used for times?

Reply:
Made that change revert it back if that's not the case